data/data-files/server-side-eval/rollout-or-experiment.yml

```diff
@@ -0,0 +1,154 @@
+---
+name: rollout or experiment - <NAME>
+
+# The test cases in this file cover very basic behavior of rollouts and experiments in terms of
+# iterating through the variations and returning one of them. We don't care what the exact bucket
+# value is here-- the weights have been set up so that the tests will work as long as the bucket
+# value is not zero. The detailed tests for the bucket value computation are implemented in Go
+# code in server_side_eval_bucketing.go.
```

Comment:
This file is the same as the equivalent one in https://github.com/launchdarkly/sdk-test-harness/pull/56 except that I changed the name "user" to "context". As the comment says, the specific bucket value doesn't matter for these particular tests, so the context attributes are arbitrary.